database/src/main/java/com/firebase/ui/database/FirebaseRecyclerAdapter.java

```diff
@@ -110,9 +110,25 @@ public void onChanged(EventType type, int index, int oldIndex) {
             public void onCancelled(DatabaseError databaseError) {
                 FirebaseRecyclerAdapter.this.onCancelled(databaseError);
             }
+
+            @Override
+            public void onReady() {
+                FirebaseRecyclerAdapter.this.onDataChanged();
```

Comment:
Agree, might as well change the signature now while it's still private and get the name you actually want.